backend/kernelCI_app/views/buildTestsView.py

```diff
@@ -1,48 +1,25 @@
 from django.http import JsonResponse
 from django.views import View
-from django.db.models import Q, TextField, Sum, Count, Case, When, F, Func, Value, Min
-from django.db.models.functions import Concat
-from django.contrib.postgres.aggregates import ArrayAgg
 
 from kernelCI_app.models import Tests
-from kernelCI_app.serializers import BuildTestsSerializer
 
 
 class BuildTests(View):
 
     def get(self, request, build_id):
```

Comment:
Is the `request` param being used?

Reply:
No, but I am forced to accept it in order to also accept the `build_id`, as it is the third argument of the function.